revup/revup.py

```diff
@@ -199,6 +199,13 @@ async def github_connection(
             f"the same repo as the remote {args.remote_info}."
         )
 
+    if not args.github_oauth:
+        args.github_oauth = await git_ctx.credential(
+            protocol="https",
+            host=args.github_url,
+            path=f"/{fork_info.owner}/{fork_info.name}.git",
```

Comment:
The leading "/" here is incorrect btw, and fails for credential helpers that do naive string matching on URL path 😞.  It should be just `path=f"{fork_info.owner}/{fork_info.name}.git"`

Reply:
it's been working for me despite this but i guess there's some subtlety about which credential helpers are being used?

Reply:
https://github.com/Skydio/revup/pull/175